centreon/tests/php/Core/Security/Vault/Domain/Model/NewVaultConfigurationTest.php

```diff
@@ -34,7 +34,10 @@
 for ($index = 0; $index <= NewVaultConfiguration::MAX_LENGTH; $index++) {
     $invalidMaxLengthString .= 'a';
 }
-
+$invalidNameMaxLengthString = '';
+for ($index = 0; $index <= NewVaultConfiguration::NAME_MAX_LENGTH; $index++) {
+    $invalidNameMaxLengthString .= 'a';
+}
```

Comment:
The use of [str_repeat](https://www.php.net/manual/fr/function.str-repeat.php) is probably easier to understand ?
```suggestion
$invalidNameMaxLengthString = str_repeat('a', 1 + NewVaultConfiguration::NAME_MAX_LENGTH);
```